libs/platform/experience/src/routes.ts

```diff
@@ -18,13 +18,38 @@ export const defaultExperienceRoutes: RouteConfig[] = [
         .resolveToken('USER.AUTHENTICATED')
         .pipe(
           take(1),
-          map((state) => {
```

Comment:
Don't get it why we are changing current behaviour - it's a bug fix?

Reply:
the behaviour is the same we updated lit-router too to support this one